### DISCUSSION TITLE
Request: Focalboard LXC

### DISCUSSION THREAD

**Discussion by Kyngo** [tteck/Proxmox#3222] [Ideas]
Some people want to use Notion-like applications and I think Focalboard is an awesome open source alternative, it would be great to have it as a helper script app :)

**Comment by Mellowlynx** [comment]
It would be nice, indeed

**Comment by OddSquirrel** [comment]
Yep, this one seems convoluted enough to warrant a script for sure. 😎

**Comment by Mellowlynx** [comment]
It's not that convoluted, just deploy a Docker LXC and run from its console `docker run -it -p 80:8000 mattermost/focalboard`,
this will get you started.
But I would map the focalboard data folder to a local one, so the data is safe for updates, make it restart, etc., so I executed:
```
docker run -d --restart=always -it -p 80:8000 -v fbdata:/opt/focalboard/data mattermost/focalboard
```

**Reply by OddSquirrel** [reply]
Yes, Docker is most often the easiest solution, which is why I tend to prefer it. But I thought this place was for people with an inherent Docker allergy, bent on direct LXC installation, so I was referring to the regular [server installation](https://www.focalboard.com/download/personal-edition/ubuntu/) process.

I may be wrong, but I, personally, don't see much sense in spinning up an LXC with Docker for an app that could just as well be run with a three-liner via Portainer.

**Reply by Mellowlynx** [reply]
I get that, but ttack can't do everything at once, so if you already have a docker LXC, it might just an idea to use that for now.
If you can wait, fine, it might be added, it might not be.

I also get the Docker allergy, kind of…, I have one LXC that runs docker for the smaller/lesser used services I have.
Happier with that, than 5 individual LXC's that use more resources overall.
It's personal, definitely, but sometimes getting outside your convert lever is the better over all option and might keep it simpler.
